server/search.js

```diff
@@ -1,8 +1,8 @@
-SearchSource.defineSource('orders', function (searchText) {
-  const options = { limit: 10 };
-  if (searchText) {
-    const selector = {shopifyOrderNumber: parseInt(searchText, 10)};
-    return Orders.find(selector, options).fetch();
-  }
-  return ReactionCore.Collections.Orders.find({}, options).fetch();
-});
+// SearchSource.defineSource('orders', function (searchText) {
```

Comment:
I think we can just delete this file.

We should still think through a better solution for search for admins that doesn't involve subscribing to every order all the time.